SubmittedAssignments/Arti_Kadam/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style2.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+    <title>Visiting Card</title>
+</head>
+<body>
+    <center style="padding-top:40px">
+        <div class="card1-wrapper card-front">
+            <!--card background-->
+            <div class="card1-background">
+                <div class="bottomrect1"></div>
+                <div class="bottomrect2"></div>
+                <div class="bottomrect3"></div>
+            </div>
+
+
+        <div class="card2-wrapper card-back">
```

Comment:
**card2-wrapper card-back** class name should be like this **card-wrapper card-back**